package/src/components/MessageInput/__tests__/__snapshots__/FileUploadPreview.test.js.snap

```diff
@@ -1059,8 +1059,8 @@ Array [
     onChange={[Function]}
     snapPoints={
       Array [
-        288,
-        1314,
+        308,
```

Comment:
Let's apply DRY for values 308 and 1334 and create constants with good names. It will increase readability and be used throughout the project and changed in one place.

Reply:
it's a snapshot nvm